.planning/phases/33-nixlbench-overview-and-build/33-01-PLAN.md

```
@@ -0,0 +1,279 @@
+- Common Meson options table (subset):
+
+  | Option | Description | Default |
+  |--------|-------------|---------|
+  | `nixl_path` | Path to NIXL installation | `/usr/local` |
+  | `buildtype` | Build type: `debug`, `release`, `debugoptimized` | `release` |
+  | `prefix` | Installation prefix | `/usr/local` |
+  | `etcd_inc_path` | Path to etcd C++ client headers | `""` |
+  | `etcd_lib_path` | Path to etcd C++ client library | `""` |
+
+- Post-install environment setup:
+  ```bash
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)